projects/ngx-skeleton-loader/src/lib/ngx-skeleton-loader.component.spec.ts

```diff
@@ -202,9 +202,11 @@ describe('NgxSkeletonLoaderComponent', () => {
           '.skeletons-with-theming .skeleton-loader.circle',
         ).attributes as NamedNodeMap;
 
-        expect((skeletonWithTheming.getNamedItem('style') as Attr).value).toBe(
-          'width: 70px; height: 70px; border-radius: 10px;',
-        );
+        const style = (skeletonWithTheming.getNamedItem('style') as Attr).value;
+        // These values can be in any order.
```

Comment:
The previous test was written that way intentionally. So that, the assertion should check if the values were added in that order - and only these values in specific.

Can you revert that, please?

Reply:
They’re in another order now.